thrimshim/thrimshim/main.py

```diff
@@ -408,6 +413,7 @@ def normalize(value):
 		return 'Sheet columns have changed since editing has begun. Please review changes\n' + changes, 409
 
 	if new_row['state'] == 'MODIFIED':
+		missing = []
```

Comment:
Drive-by fix here for a bug I randomly spotted.
Previously we didn't define `missing` before using it here, but there happened to be a `missing` defined previously which would always be empty, so as long as no columns were ever missing it would still work.